Site Docs: Add Details To Alert Creation Process

harding opened the pull request:
This is, I think, the last thing left on the todo list from #934

Preview: https://github.com/harding/bitcoin.org/tree/alert-docs#alerts

saivann commented on README.md:
```diff
@@ -535,34 +535,148 @@ is the particular version:
+- Saïvann Carignan, [@saivann](https://github.com/saivann), saivann on Freenode
+- Dave Harding, [@harding](https://github.com/harding), harding on Freenode
+- Wladimir van der Laan, [@laanwj](https://github.com/laanwj), wumpus on Freenode
+- Theymos, [@theymos](https://github.com/theymos), theymos on Freenode
```
I noticed @theymos's email address isn't on his GitHub profile as suggested above. @theymos, is it possible for you to add the information, just in case?

theymos replied:
Done.

harding replied:
Thanks!

saivann commented on README.md:
```diff
@@ -535,34 +535,148 @@ is the particular version:
+#### Basic Alert
+
+1. Open your editor on a file named `_alerts/YYYY-MM-DD-short-title.md`
+   (the alert will appear as <https://bitcoin.org/YYYY-MM-DD-short-title>).
```
@harding Found a mistake here! The url will actually be https://bitcoin.org/en/alert/YYYY-MM-DD-short-title

saivann commented on README.md:
```diff
@@ -535,34 +535,148 @@ is the particular version:
+    ## Title displayed on alert page
+    title: "11/12 March 2013 Chain Fork"
+    ## Short URL for use in P2P network alerts: https://bitcoin.org/<alias>
+    alias: "chainfork"
```
@harding Would it be perhaps more self-explanatory to rename this variable to "shorturl" everywhere?

saivann commented:
@harding Except for the comments above, c014e92 LGTM, thanks!

harding commented:
@saivann thanks for catching that mistake, and making that improvement suggestion.  Here's what I've done:

- Rebased to fix a merge conflict
- Fixed the error
- Changed `alias` to `shorturl`
- Performed a full build and HTML diff to confirm that all the redirect pages were created at the same locations and that nothing else changed

saivann commented:
Untested LGTM, thanks!

harding commented:
@saivann thanks!

In the absence of critical feedback, this PR will be merged Tuesday.
